src/EventTimer.spec.ts

```diff
@@ -109,8 +109,8 @@ describe('EventTimer', () => {
 		const eventTimer = EventTimer.get();
 		eventTimer.mark(MARK_NAME);
 		expect(eventTimer.events).toHaveLength(1);
-		expect(eventTimer.events[0].name).toBe(MARK_NAME);
-		expect(eventTimer.events[0].ts).toBe(1);
+		expect(eventTimer.events[0]?.name).toBe('mark_name');
```

Comment:
Cheeky merging from `main`!

```suggestion
		expect(eventTimer.events[0]?.name).toBe(MARK_NAME);
```